tests/python/openassetio/pluginSystem/test_pluginsystemmanagerfactory.py

```diff
@@ -29,7 +29,7 @@
 import pytest
```

Comment:
[fix] File needs renaming and method names

Reply:
Fixed in 3e1b27bb

Reply:
Maybe 'whole word' leading you astray?